test/utils/transclude-directive.spec.js

```diff
@@ -0,0 +1,61 @@
+describe('Directive:  pfTransclude', function () {
+  var $scope;
+  var $compile;
+  var outerId;
+  var directiveId;
+  var childId;
+  var parentId;
+
+  // load the controller's module
+  beforeEach(function () {
+    module('patternfly.utils');
+    module('patternfly.testutils');
+  });
+
+  beforeEach(inject(function (_$compile_, _$rootScope_) {
+    $compile = _$compile_;
+    $scope = _$rootScope_;
+  }));
+
+  var compileHTML = function (transludeDirective, scope) {
+    var markup = '<div ng-controller="TestUtilCtrl" >' +
+      '  <input id="outerId" value="{{$id}}">{{$id}}</input>' +
+      '  <div ' + transludeDirective + ' class="pf-transclude-example">' +
+      '    <input id="childId" value="{{$id}}">' +
+      '    <input id="parentId" value="{{$parent.$id}}">' +
+      '  </div>' +
+      '</div';
+    var element = $compile(angular.element(markup))(scope);
+    scope.$apply();
+    scope.$digest();
```

Comment:
Left over from trying to get things digested and working correctly. Removed the $apply call.